test/integration/apiserver/admissionwebhook/admission_test.go

```diff
@@ -1728,50 +1728,52 @@ func createV1MutationWebhook(client clientset.Interface, endpoint, convertedEndp
 //
 //	go run generate_cert.go  --rsa-bits 2048 --host 127.0.0.1,::1,example.com --ca --start-date "Jan 1 00:00:00 1970" --duration=1000000h
```

Comment:
I assume you regenerated this with a new domain, right?
if that is the case please update the comment to reflect the actual command used

Reply:
Done.